guby_backend/views.py

```diff
@@ -28,23 +31,29 @@ def get_rolename(request, id):
         "role_name" : role_name
     }
 
+@login_required(login_url='/app/users/login/')
```

Comment:
Isn't there a central location to provide the login_url instead of at each decorator? Change would become expensive later otherwise

Reply:
Looks like there's a way to specify it in `settings.py`. Can we change these over to use that instead? https://docs.djangoproject.com/en/3.0/ref/settings/#std:setting-LOGIN_URL